ingestion-beam/src/main/java/com/mozilla/telemetry/options/SinkOptions.java

```diff
@@ -108,6 +109,14 @@
 
   void setOutputFileFormat(OutputFileFormat value);
 
+  @Description("Row format for --outputType=bigquery; must be one of"
+      + " raw (each row contains payload[Bytes] and attributes as top level fields) or"
+      + " decoded (each row contains payload[Bytes] and attributes as nested metadata fields) or"
+      + " payload (each row is extracted from payload); defaults to payload")
+  ValueProvider<TableRowFormat> getOutputTableRowFormat();
```

Comment:
reflagging for review because I added a format and changed the format names to be more like the pubsub topic names.